src/js/components/account/profile.js

```diff
@@ -0,0 +1,18 @@
+import { BasicBlock } from "../basic-block";
+
+export const ProfileInfo = (profile) => {
```

Comment:
beware of creating a component that's used once - this is a very specific thing and it probably makes more sense having this code on the page class than in a dedicated component until it's clearly used in multiple places. All it means right now is another file to look at when working on the profile page.

Reply:
https://github.com/OpenUpSA/muni-portal-frontend/issues/69